packages/big-design/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './components';
+export * from './mixins';
```

Comment:
Looks good, can you change the commit to either a `feat` or `fix` (not sure how you want to look at this), since `chore` commits don't generate changelog entries.